sync/core.extension.yml

```diff
@@ -50,7 +49,6 @@ module:
   monolog: 0
   node: 0
   options: 0
-  page_cache: 0
```

Comment:
This module is also responsible for cache. Removing this configuration will uninstall the modules from the production instance.